docs/src/docs/secret_deplyment.md

```diff
@@ -44,6 +44,36 @@ metadata:
 type: Opaque
 ```
 
+```
+Name:         sonarcloud
+Namespace:    security-controls
+Type:  Opaque
+
+Data
+====
+SONARCLOUD_API_KEY:  SonarCloud API key
+```
+
+```
+Name:         github
+Namespace:    security-controls
+Type:  Opaque
+
+Data
+====
+GITHUB_TOKEN:  GitHub personal access token
+```
+
+```
+Name:         codacy
+Namespace:    security-controls
+Type:  Opaque
+
+Data
+====
+CODACY_API_KEY:  Codacy API key
+```
+
```

Comment:
Is there any reason why SONARCLOUD_API_KEY, GITHUB_TOKEN, and CODACY_API_KEY are not in the same secret file `security-controls`?

Reply:
All `security-controls` namespace secrets are in the same file. Looks like you put these 3 in different files.

Reply:
Can we please add these secrets following the documentation format?

Reply:
Easier rotation and cleaner separation. Yes, I can follow the format.